ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml

```diff
@@ -91,10 +101,10 @@ tests:
     env:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "9"
-      COMPUTE_NODE_TYPE: m5.2xlarge
+      COMPUTE_NODE_TYPE: m6i.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
       LOKI_USE_SERVICEMONITOR: "false"
-      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
-      SET_ENV_BY_PLATFORM: custom
```

Comment:
Only when this is set to custom, the overwritten of OPENSHIFT_INFRA_NODE_INSTANCE_TYPE is used. https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh#L639
cc: @liqcui

Reply:
+1
Yes, need to specify SET_ENV_BY_PLATFORM: custom.